test/functional/wallet_bumpfee.py

```diff
@@ -374,7 +375,7 @@ def test_segwit_bumpfee_succeeds(self, rbf_node, dest_address):
 def test_nonrbf_bumpfee_fails(self, peer_node, dest_address):
     self.log.info('Test that we cannot replace a non RBF transaction')
```

Comment:
In fa16051eac9a4bc1a7e0234b65d615e655ff7cf0 "rpc: Allow fullrbf fee bump"

The test name and logging should be updated to reflect the new behavior of the test.